kernel/src/net/socket/netlink/kobject_uevent/message/uevent.rs

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: MPL-2.0
+
+// FIXME: The code in this file is currently unused
+// because we haven't implemented support for the uevent inode in sysfs yet.
```

Comment:
> However, due to certain bugs in sysfs, I am currently unable to add inodes related to devices in the systree.

Shouldn't we fix the bugs first?

Reply:
After some refactoring, the original warnings disappear. So I remove the `#![expect(dead_code)]` line and its associate comments. But TBH, I don't know what happens. The code in this file should still be dead code.